GOVERNANCE.rst

```diff
@@ -251,12 +258,16 @@ Appointment
 
 Membership of the CC is by nomination by a core developer and a vote by
 all core developers. A nomination will result in discussion which cannot
-take more than a week and then a vote by the core developers which will
-stay open for a week. CC membership votes are subject to:
+take more than 5 days excluding weekends and then a vote by the core
```

Comment:
hm, this allows discussions that take 2 seconds - who decides when the discussion ends?

I would say:
* proposer should make a case
* discussion should be open for some mimimum time period